### ISSUE TITLE
beach POI symbol

### ISSUE THREAD

**Issue by hakuchi** [osmandapp/Osmand#3198]
the beach show some strange looking POI symbol.
I think it is enough to show only the area, because to many POI symbols make the map more confusing.

![pic](http://fs5.directupload.net/images/161023/ot3minmi.png)

**Comment by xmd5a2** [comment]
That's very strange. I thought this bug was solved years ago. Can't reproduce. Rendering style uses "beach" icon
```xml
<case minzoom="16" tag="natural" value="beach" icon="beach" iconOrder="79"/>
```
But icon on screenshot is old beach shader which is named "beach_old". Checked apk contents.

**Comment by hakuchi** [comment]
Screenshot is from PlayStore OsmAnd+ Beta 2.4.7

Made today a complete new installation from PlayStore(now v 2.5), but POI symbol is the same.

If I change the line in my default.rendere.xml(is the same as the file here on GitHub) on my mobile phone to
`<case minzoom="16" tag="natural" value="beach" icon="something other" iconOrder="79"/>"`
no POI symbol is displayed.

If I change to
`<case minzoom="16" tag="natural" value="beach" icon="beach" iconOrder="79"/>`
the Poi symbol from screenshot in displayed.

Same strange POI symbol is on zoo:
![pic](http://img5.fotos-hochladen.net/uploads/screenshot20164tpbkgzlir.png)

**Comment by hakuchi** [comment]
same in the OsmAnd+ 2.5.1 from PlayStore with latest maps

**Comment by hakuchi** [comment]
same in the OsmAnd+ 2.5.3 from PlayStore with latest maps

**Comment by sonora** [comment]
Looks like in your setup, the icon displays not what we define as the icon, but uses the "shader" (which we use to paint the background). I assume you may see this for more area features with shader background where we display an additional icon.

But I cannot reproduce so far ...  Loos like screenshot 1 above is from a Germany_Bavaria map. What date/version is it? Have you updated the map lately and tried again?

**Comment by hakuchi** [comment]
I have made yesterday a complete new installation from OsmAnd.

Now I have OsmAnd+ 2.5.3(beta) from PlayStore with latest maps from Germany Bavaria.
The Symbol from beach and zoo are the same as above.
![beach pic](http://img5.fotos-hochladen.net/uploads/screenshot201640ux5paw8i.png)

**Comment by hakuchi** [comment]
I deleted the version from PlayStore and installed the latest nightly from OsmAnd homepage.
In this build(2.5.0#15377D) the beach and zoo symbols are correct.

The version from Playstore(+ and free)(2.5.3) always display the wrong POI symbol.

**Comment by sonora** [comment]
Even if you now uninstall the nightly and go back to a Play Store version?

**Comment by hakuchi** [comment]
I installed in this way:
> 
> 1. uninstalled PlayStore version  
>    also deleted osmand folder from SD
> 2. installed nightly
> 3. downloaded data and latest map   
>    POI symbol is displayed as like   
>    https://github.com/osmandapp/OsmAnd-resources/blob/master/rendering_styles/style-icons/drawable-hdpi/mm_beach.png
>    and    
>    https://github.com/osmandapp/OsmAnd-resources/blob/master/rendering_styles/style-icons/drawable-hdpi/mm_zoo.png
> 4. uninstalled nightly     
>    not deleted osmand files from SD!!
> 5. installed latest from PlayStore
> 6. no data and map download necessary,   
>    but the POI symbol displayed as in the screenshots above.

So nothing changed

**Comment by hakuchi** [comment]
now i extracted the data from latest osmand.apk and from the nightly.apk with winzip. I have compared the files and found differences.

In the version from PlayStore are present some files what not existing in nightly.

![differences](http://img5.fotos-hochladen.net/uploads/osmandf610yl2x8g.jpg)
also _h_zoo.png_ is not present in the nightly.

Not all 369 differences must be wrong, they can also be some new symbols/icons.
So I think there is some error in the way how the PlayStore version is build, because I can not find a _h_beach.png_ and _h_zoo.png_ here on GitHub

**Comment by xmd5a2** [comment]
We already found source of issue. Will be fixed in next minor release.

**Comment by hakuchi** [comment]
thx
is fixed in 2.5.4
